src/birthder.js

```diff
@@ -428,6 +440,50 @@
       return
     }
 
+    /**
+     * Daily checker.
+     *
+     * @param {Robot} robot - Hubot instance.
+     * @return {void}
+     */
+    const birthdaySetReminder = robot => {
```

Comment:
We've already got the function which name includes the word 'reminder'. Let's name this function `detectBirthdaylessUsers`.